qontract_utils/qontract_utils/kubernetes/__init__.py

```
@@ -0,0 +1,49 @@
+    "TIMEOUT",
+    "AlreadyExistsError",
+    "Namespace",
+    "ForbiddenError",
+    "KubernetesApi",
+    "KubernetesApiCallContext",
+    "KubernetesApiError",
+    "Namespace",
+    "NotFoundError",
+    "UnauthorizedError",
+    "from_api_error",
+]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)